### PR TITLE
update basic features and experimental features for v6.2

### PR DESCRIPTION
**Pull request by shichun-0415**
### What is changed, added or deleted? (Required)
Update basic features and experimental features for v6.2.0. The changes are listed as follows:

- [ALTER TABLE ... COMPACT](https://github.com/pingcap/docs-cn/blob/f1911304715cdbe8d96194529c1a8d57c8ee5eca/sql-statements/sql-statement-alter-table-compact.md): experimental -> GA
- [前台限流](https://github.com/pingcap/docs-cn/blob/f1911304715cdbe8d96194529c1a8d57c8ee5eca/tikv-configuration-file.md#%E5%89%8D%E5%8F%B0%E9%99%90%E6%B5%81): experimental -> GA
- 在单条语句中添加多列: experimental -> GA
- [使用线程池处理存储引擎的读请求](https://github.com/pingcap/docs-cn/blob/f1911304715cdbe8d96194529c1a8d57c8ee5eca/tiflash/tiflash-configuration.md#tiflashtoml): experimental
- [Cost Model Version 2](https://github.com/pingcap/docs-cn/blob/master/cost-model.md#cost-model-version-2): experimental
- [Fast Mode](https://github.com/pingcap/docs-cn/blob/master/develop/dev-guide-use-fastscan.md#fast-mode): experimental 
- [后台限流](https://github.com/pingcap/docs-cn/blob/master/tikv-configuration-file.md#%E5%90%8E%E5%8F%B0%E9%99%90%E6%B5%81): experimental 
- [磁盘资源配额](https://github.com/pingcap/docs-cn/blob/f1911304715cdbe8d96194529c1a8d57c8ee5eca/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#%E7%A3%81%E7%9B%98%E8%B5%84%E6%BA%90%E9%85%8D%E9%A2%9D-%E4%BB%8E-v620-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5): experimental
- [DM 增量数据校验](https://github.com/pingcap/docs-cn/blob/f1911304715cdbe8d96194529c1a8d57c8ee5eca/dm/dm-continuous-data-validation.md): experimental
- [RawKV 跨集群复制](https://github.com/pingcap/docs-cn/blob/f1911304715cdbe8d96194529c1a8d57c8ee5eca/tikv-configuration-file.md#api-version-%E4%BB%8E-v610-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5): experimental
- [备份和恢复 RawKV 数据](https://github.com/pingcap/docs-cn/blob/master/br/rawkv-backup-and-restore.md)：废弃，从实验特性移除

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: experimental-features.md [hunk 1]
```diff
@@ -13,13 +13,18 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
 + [支持收集部分列的统计信息](/statistics.md#收集部分列的统计信息)（v5.4 实验特性）
 + [支持统计信息的同步加载](/statistics.md#统计信息的加载)（v5.4 实验特性）
 + [限制 ANALYZE 的内存使用量](/statistics.md#统计信息收集的内存限制) (v6.1.0 实验特性)
++ [使用线程池处理存储引擎的读请求](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml)（v6.2.0 实验特性）
++ [Cost Model Version 2](/cost-model.md#cost-model-version-2)（v6.2.0 实验特性）
++ [Fast Mode](/develop/dev-guide-read-in-fast-mode.md)（v6.2.0 实验特性）
```
Comment (shichun-0415) [inline experimental-features.md:18]: Feature name and link to be updated

**Comment by TomShawn** [comment]
LGTM

File: experimental-features.md (hunk 1 above)
Comment (hfxsd) [inline experimental-features.md:18]:
```suggestion
+ [Fast Scan](/develop/dev-guide-read-in-fast-mode.md)（v6.2.0 实验特性）
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: experimental-features.md [hunk 2]
```diff
@@ -39,13 +43,15 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
...
 + TiKV 引入 [API v2](/tikv-configuration-file.md#api-version-从-v610-版本开始引入)。（v6.1.0 实验特性）
 
-## 备份与恢复
-
-+ [RawKV 备份和恢复](/br/rawkv-backup-and-restore.md)。（v3.1 实验特性）
-
 ## 数据迁移
 
 + [使用 WebUI](/dm/dm-webui-guide.md) 管理 DM 迁移任务。（v6.0 实验特性）
++ [磁盘资源配额](/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#磁盘资源配额-从-v620-版本开始引入)（v6.2.0 实验特性）
```
Comment (ran-huang) [inline experimental-features.md:49]:
```suggestion
+ 为 TiDB Lightning 设置[磁盘资源配额](/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#磁盘资源配额-从-v620-版本开始引入)（v6.2.0 实验特性）
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: experimental-features.md [hunk 3]
```diff
@@ -13,12 +13,17 @@ aliases: ['/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-f
 + [支持收集部分列的统计信息](/statistics.md#收集部分列的统计信息)（v5.4 实验特性）
 + [支持统计信息的同步加载](/statistics.md#统计信息的加载)（v5.4 实验特性）
 + [限制 ANALYZE 的内存使用量](/statistics.md#统计信息收集的内存限制) (v6.1.0 实验特性)
++ [使用线程池处理存储引擎的读请求](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml)（v6.2.0 实验特性）
++ [Cost Model Version 2](/cost-model.md#cost-model-version-2)（v6.2.0 实验特性）
++ [FastScan](/develop/dev-guide-read-in-fast-mode.md)（v6.2.0 实验特性）
```
Comment (hfxsd) [inline experimental-features.md:18]:
```suggestion
+ [FastScan](/develop/dev-guide-use-fastscan.md)（v6.2.0 实验特性）
```

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by Frank945946 — APPROVED** [review]

**Review by songrijie — APPROVED** [review]
LGTM

**Review by joey-yez — APPROVED** [review]
LGTM for these 3 features:ALTER TABLE ... COMPACT
使用线程池处理存储引擎的读请求
FastScan

**Comment by easonn7** [comment]
TiKV 前台限流，TiKV 后台限流，在单条语句中添加多列
这三条 LGTM

**Pull request merged**